app/views/events/show.erb

```diff
@@ -0,0 +1,28 @@
+<div class="navbar navbar-static-top navbar-default">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="/sources/<%= @client.identifier %>">Dashboard</a>
+    </div>
+    <div class="navbar-collapse collapse">
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="/sources/<%= @client.identifier%>/urls">Urls</a></li>
+        <li><a href="/sources/<%= @client.identifier%>/events">Events</a></li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+<div class="container">
+  <h1>24 Hour Breakdown - <%= @event.event_name  %></h1>
+  <table id="events-table" class="table text-center">
+    <% (1..24).to_a.each do |hour| %>
```

Comment:
this is too much logic in your view. you should set `(1..24).to_a` as a variable and pass it in from the controller